lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -59,13 +72,55 @@ def export_kick_off(klass)
           versions :stu3
```

Comment:
Bulk data requires `r4` because async doesn't in exist prior versions.

Reply:
Both Smart-health reference server and BCDA test server are STU3.

Reply:
But `prefer: respond-async` isn't standardized until `R4`.   I guess you could argue that they are STU3 servers doing extra stuff.  Regardless of whether it is technically valid in STU3, this definitely should be applicable to R4, R5, R6+++, so just remove that line altogether so that it isn't tied to one version.